Context/Context.php

```diff
@@ -61,48 +105,53 @@ public function get(ContextKey $key)
             return $this->value;
         }
         if (null === $this->parent) {
-            throw new ContextValueNotFoundException();
+            throw new ContextValueNotFoundException('HUH');
```

Comment:
This was left from our debugging; we could probably change it to something more meaningful.